src/item.rb

```diff
@@ -23,4 +23,14 @@ def add_label(label)
     @label = label
     label.items.push(self) unless label.items.include?(self)
   end
+
+  def move_to_archive?
+    return @archived = true if can_be_achieved?
+  end
+
+  private
+
+  def can_be_archieved?
+    @publish_date > 10
+  end
```

Comment:
- [] Kindly check the requirements that say that publish_date is a date Class, so you may need to parse this class to check the difference between the years and today's date. Now you are comparing aS string with a number.
https://ruby-doc.org/stdlib-2.5.1/libdoc/date/rdoc/Date.html
![image](https://user-images.githubusercontent.com/103154117/218784995-7f6b2755-cdec-47ac-a7f6-99684b161c52.png)

![image](https://user-images.githubusercontent.com/103154117/218784824-71373407-a265-4543-8c2c-c1587200af63.png)